src/Client/pages/WishList.fs

```diff
@@ -90,14 +91,14 @@ let update (msg:WishListMsg) model : Model*Cmd<WishListMsg> =
     | FetchError _ -> 
         model, Cmd.none
 
-let addNewBookForm (model:Model) dispatch =
+let newBookForm (model:Model) dispatch =
     let buttonActive = if String.IsNullOrEmpty model.NewBook.Title || String.IsNullOrEmpty model.NewBook.Authors then "btn-disabled" else "btn-primary"
     
-    let titleStatus = if String.IsNullOrEmpty model.NewBook.Title then "" else "has-feedback has-success"
```

Comment:
``has-feedback`` was used in each case further below anyway where this data is consumed.  Perhaps it should be removed from the places below?